bundles/org.openhab.binding.androidtv/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -134,6 +134,99 @@
 				<default>6466</default>
 				<advanced>true</advanced>
 			</parameter>
+			<parameter name="keystoreFileName" type="text">
+				<label>Keystore File Name</label>
+				<description>Java keystore containing key and certs</description>
+			</parameter>
+			<parameter name="keystorePassword" type="text">
+				<context>password</context>
+				<label>Keystore Password</label>
+				<description>Password for the keystore file</description>
+			</parameter>
+			<parameter name="reconnect" type="integer" min="0">
+				<label>Reconnect Delay</label>
+				<description>Delay between reconnection attempts</description>
+				<default>60</default>
+			</parameter>
+			<parameter name="heartbeat" type="integer" min="0">
+				<label>Heartbeat Frequency</label>
+				<description>Frequency of heartbeats</description>
+				<default>5</default>
+			</parameter>
+			<parameter name="delay" type="integer" min="0">
+				<label>Delay</label>
+				<description>Delay between messages</description>
+				<default>0</default>
+			</parameter>
+		</config-description>
+
+	</thing-type>
+
+	<thing-type id="philipstv">
+		<label>Philips TV</label>
+		<description>A Philips TV device</description>
+
+		<channels>
+			<channel id="debug" typeId="debug"/>
+			<channel id="keypress" typeId="keypress"/>
+			<channel id="keyboard" typeId="keyboard"/>
+			<channel id="keycode" typeId="keycode"/>
+			<channel id="pincode" typeId="pincode"/>
+			<channel id="volume" typeId="system.volume"/>
+			<channel id="mute" typeId="system.mute"/>
+			<channel id="power" typeId="system.power"/>
+			<channel id="brightness" typeId="brightness"/>
+			<channel id="contrast" typeId="contrast"/>
+			<channel id="sharpness" typeId="sharpness"/>
+			<channel id="app" typeId="app"/>
+			<channel id="appname" typeId="appname"/>
+			<channel id="appicon" typeId="appicon"/>
+			<channel id="tvChannel" typeId="tvChannel"/>
+			<channel id="player" typeId="player"/>
+			<channel id="searchContent" typeId="searchContent"/>
+			<channel id="ambilightPower" typeId="ambilightPower"/>
+			<channel id="ambilightHuePower" typeId="ambilightHuePower"/>
+			<channel id="ambilightLoungePower" typeId="ambilightLoungePower"/>
+			<channel id="ambilightStyle" typeId="ambilightStyle"/>
+			<channel id="ambilightColor" typeId="ambilightColor"/>
+			<channel id="ambilightLeftColor" typeId="ambilightLeftColor"/>
+			<channel id="ambilightRightColor" typeId="ambilightRightColor"/>
+			<channel id="ambilightTopColor" typeId="ambilightTopColor"/>
+			<channel id="ambilightBottomColor" typeId="ambilightBottomColor"/>
```

Comment:
I'm concerned this will be a breaking change for the community.  The Philips binding that was pulled into here has a large community user base.  While I realize that it wasn't an official binding, I am concerned by just changing a ton of channels.  I don't disagree that they should maintain the approved naming convention, just don't wont to break everyone's working configs.  Thoughts?

Reply:
It is already a breaking change for those PhilipsTV users as they have to use a different binding and update their config accordingly. Replacing old/ new channel id’s would be simple.

FTR: only talking about the new channels, not existing shield tv etc

Reply:
I've been thinking about this and I have concerns that the differing naming convention could cause confusion by the user's. Also, I can't change all of them in Philips as there is overlap across protocols which would break. Not the vendor specific ones, but the more generic ones.

Reply:
I understand. Lets stick to the existing naming convention within the binding.

Reply:
Thank you!